tapfreighter/chain_porter.go

```diff
@@ -994,7 +994,7 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 				"address parcel")
 		}
 		fundSendRes, err := p.cfg.AssetWallet.FundAddressSend(
-			ctx, addrParcel.destAddrs...,
+			ctx, true, addrParcel.destAddrs...,
```

Comment:
I think we can make this diff/change and future call sites more readable if we introduce an enum early here instead of a `bool`. 

It could be something like (random names, can def be improved): 
```go
type CoinSelectType uint8

const (
    DefaultCoinSelect CoinSelectType
    Bip86Only
    ScriptTreesAllowed 
)
```

Then we can make `DefaultCoinSelect = Bip86Only` when parsing, which is typically the behavior you want unless doing something fancy.

Reply:
Makes sense! Changed the type. But had to keep the default value at `ScriptTreesAllowed` to not break existing users of the `FundVirtualPsbts` RPC call (which is primarily used for doing script sends, at least in our itests).